cmd/create.go

```diff
@@ -12,20 +19,53 @@ var createCmd = &cobra.Command{
 	Short: "Create an admin user",
 	Long:  `Create an admin user.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("TODO")
+		apiUrl, _ := cmd.Flags().GetString("fga-api-url")
+		apiToken, _ := cmd.Flags().GetString("fga-api-token")
+		storeId, _ := cmd.Flags().GetString("fga-store-id")
+		modelId, _ := cmd.Flags().GetString("fga-model-id")
+		user, _ := cmd.Flags().GetString("user")
+
+		createAdmin(apiUrl, apiToken, storeId, modelId, user)
 	},
 }
 
 func init() {
 	adminCmd.AddCommand(createCmd)
 
-	// Here you will define your flags and configuration settings.
+	createCmd.Flags().String("fga-api-url", "", "The openfga API URL")
+	createCmd.Flags().String("fga-api-token", "", "The openfga API token")
+	createCmd.Flags().String("fga-store-id", "", "The openfga store")
+	createCmd.Flags().String("fga-model-id", "", "The openfga model")
+	createCmd.Flags().String("user", "", "The admin user name, if not specified an autogenerated ID will be used")
+	createCmd.MarkFlagRequired("fga-api-url")
+	createCmd.MarkFlagRequired("fga-api-token")
+	createCmd.MarkFlagRequired("fga-store-id")
+}
+
+func createAdmin(apiUrl, apiToken, storeId, ModelId, user string) {
+	logger := logging.NewNoopLogger()
+	tracer := tracing.NewNoopTracer()
+	monitor := monitoring.NewNoopMonitor("", logger)
+	scheme, host, err := parseURL(apiUrl)
+	if err != nil {
+		panic(err)
+	}
+	cfg := openfga.NewConfig(scheme, host, storeId, apiToken, "", false, tracer, monitor, logger)
+	fgaClient := openfga.NewClient(cfg)
+	auth := authorization.NewAuthorizer(fgaClient, tracer, monitor, logger)
 
-	// Cobra supports Persistent Flags which will work for this command
-	// and all subcommands, e.g.:
-	// createCmd.PersistentFlags().String("foo", "", "A help for foo")
+	if user == "" {
+		user = randUserName()
+	}
+
+	err = auth.CreateAdmin(context.Background(), user)
+	if err != nil {
+		panic(fmt.Errorf("failed to create user: %w", err))
+	}
```

Comment:
If the user already exists the user will get an error like this 
![image](https://github.com/canonical/identity-platform-admin-ui/assets/19745916/d1bfff88-966f-4066-a9c0-dc1ddb14e75e)

Should we handle this more gracefully?

Reply:
what about logging the error and the os.exit?